verticapy/tests/vModel/test_ridge.py

```diff
@@ -63,9 +64,9 @@ def test_features_importance(self, model):
         fim = model.features_importance()
 
         assert fim["index"] == ['alcohol', 'residual_sugar', 'citric_acid']
-        assert fim["importance"] == [65.86, 33.35, 0.79]
+        assert fim["importance"] == [52.3, 32.63, 15.07]
         # TODO: it is nicer not to have Decimal for sign
```

Comment:
You've done this TODO comment, so you can remove it.

Reply:
Did it!